src/py/crankshaft/setup.py

```diff
@@ -10,7 +10,7 @@
 setup(
     name='crankshaft',
 
-    version='0.0.1',
+    version='0.0.0',
```

Comment:
That 0.0.0 is replaced by the make file upon release. The package installed in the development environment keeps the 0.0.0, though (it is installed directly from the source code).